esmvaltool/recipes/examples/recipe_check_obs.yml

```diff
@@ -902,6 +902,19 @@ diagnostics:
     scripts: null
 
 
+  ERA5-Land:
```

Comment:
There is no need to add this check in the` recipe_check_obs.yml`. Please see issue #1909 for more information.